woocommerce/woocommerce
# Add E2E tests to create simple product in the block editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR introduces E2E tests to the simple product creation using the blocks editor.

Closes #39411.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Execute the standard commands to launch the local E2E environment.
2. Make sure the test is not flaky by running multiple instances of it repeatedly. You can do so through this command:
    ```bash
    cd plugins/woocommerce
    pnpm test:e2e-pw create-simple-product-block-editor.spec.js
    ```
4. Tests should pass.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
